python/cudf/cudf/utils/dtypes.py

```diff
@@ -233,8 +235,15 @@ def _get_nan_for_dtype(dtype: DtypeObj) -> DtypeObj:
         time_unit, _ = np.datetime_data(dtype)
         return dtype.type("nat", time_unit)
     elif dtype.kind == "f":
+        if is_pandas_nullable_extension_dtype(dtype):
+            return pd.NA
         return dtype.type("nan")
     else:
+        if (
+            is_pandas_nullable_extension_dtype(dtype)
+            and getattr(dtype, "kind", "c") in "biu"
+        ):
+            return pd.NA
```

Comment:
```suggestion
        if is_pandas_nullable_extension_dtype(dtype)
            return dtype.na_value
```

pandas ExtensionDtypes should always have this attribute

Reply:
Done 👍